src/main/java/org/primefaces/component/datatable/export/DataTableExporter.java

```diff
@@ -337,8 +342,6 @@ public void export(FacesContext context, List<DataTable> tables, OutputStream ou
      */
     protected abstract void doExport(FacesContext facesContext, DataTable table, ExportConfiguration exportConfiguration, int index) throws IOException;
 
-    public abstract String getFileExtension();
-
```

Comment:
Already declared in Interface.

Reply:
Again haven't worked on DataTableExporter, but also yes, that declaration can be removed.

Reply:
Yes, should be removed.